cmd/cache.go

```diff
@@ -19,11 +19,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"sync"
 
 	bq "cloud.google.com/go/bigquery"
 
-	"github.com/schollz/progressbar/v3"
```

Comment:
What a short life, haha.

Reply:
schollz/progressbar mantainer not supports multi line bar ...
https://github.com/schollz/progressbar/issues/6